crates/api_structs/src/community.rs

```diff
@@ -53,7 +53,7 @@ pub struct ListCommunitiesResponse {
 #[derive(Deserialize, Clone)]
 pub struct BanFromCommunity {
   pub community_id: i32,
-  pub user_id: i32,
+  pub person_id: i32,
```

Comment:
I'll make an issue to update lemmy-docs for all these changes. Its pretty easy to see them by looking at my PR on `lemmy-js-client`